CASM-3970: Change prodmgr to use the product-deletion-utility container for deletes (#15)
Cray-HPE/prodmgr: man/prodmgr.8.rst

DESCRIPTION
===========

prodmgr is the CLI for Shasta product deletion and product activation. It works by running
a generic product deletion utility container image that provides the functionality
that can delete a given version of a given product. For product activation, prodmgr

> launches a product specific utility container image that provides activation functionality.

Reviewer: ```suggestion
launches a product-specific utility container image that provides activation functionality.
```

ARGUMENTS
=========

*ACTION*

>     The action to perform on the specified product. "delete", "uninstall", or "activate". Note, both the "uninstall" and "activate" commands have been deprecated.
>     The "delete" and "uninstall" actions are identical.

Reviewer: This could be reworded to make it more immediately clear that "delete" is the new "uninstall". Something like this is better IMO:

```suggestion
    The action to perform on the specified product. Supported actions are
    "delete", "uninstall" (an alias for "delete"), and "activate". The
    "uninstall" and "activate" actions are deprecated.
```

Also note that this file hard-wraps lines for readability, which I did above.
